editor/edit-post/header/editor-actions/style.scss

```diff
@@ -0,0 +1,3 @@
+.editor-actions {
+	margin-top: -20px;
```

Comment:
Hmmm. This feels a bit like a dirty fix. 🤨😄

Reply:
Since it's not part of the radio-field-like editor switcher, it may actually make sense to have some space. Cc @jasmussen @karmatosed.

Reply:
> Hmmm. This feels a bit like a dirty fix. 🤨😄

🙈 Yeah. 

> it may actually make sense to have some space

I wondered the same. Some of this might become clearer when more consumers become interested in this space (see extensions).

Reply:
Agree with Ella, will post a mockup as a comment.